doc/classes/@GlobalScope.xml

```diff
@@ -107,6 +129,17 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="atanh">
+			<return type="float" />
+			<param index="0" name="x" type="float" />
+			<description>
+				Returns the hyperbolic arc (also called inverse) tangent of [param x], returning a value in radians. Use it to get the angle from an angle's tangent in hyperbolic space.
```

Comment:
Needs a clarification on the clamping like `asin/atan` above

Reply:
Indeed. As I mentioned above, "I want to see what people think and will update docs to reflect those special cases if agreed."

Does that mean that you think the "infinity clamping" would look as a valid solution? (I wasn't sure about this part)

Reply:
I just saw your comment on the proposal saying it looks cohesive- I will amend the patch to add the clarification of this non-standard behaviour. Thank you!

Reply:
Latest push should document this properly (I think).